src/control/server/server.go

```diff
@@ -277,21 +290,30 @@ func (srv *server) start(ctx context.Context, shutdown context.CancelFunc) error
 	sigChan := make(chan os.Signal)
 	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGQUIT, syscall.SIGTERM)
 	go func() {
-		// SIGKILL I/O Engine immediately on exit.
-		// TODO: Re-enable attempted graceful shutdown of I/O Engines.
 		sig := <-sigChan
 		srv.log.Debugf("Caught signal: %s", sig)
 
 		shutdown()
 	}()
 
 	return errors.Wrapf(srv.harness.Start(ctx, srv.sysdb, srv.pubSub, srv.cfg),
-		"%s exited with error", build.DataPlaneName)
+		"%s harness exited", build.ControlPlaneName)
 }
 
-// Start is the entry point for a daos_server instance.
+// Start is the entry point for a daos_server instance, perform the following
+// tasks in order to bring-up:
+//
+// - create server struct with component references
+// - create service wrappers for rpc and event services
+// - initialize network components
+// - initialize storage components
+// - create and add configured engine instances to server
+// - create and configure main grpc server to handle requests
+// - register actions to be taken on receipt of events
+// - start listening server and processing loops
```

Comment:
Generally speaking I'm in favor of more rather than fewer comments, particularly for unclear code. In this case, however, I'm worried that this block is likely to quickly get out of sync with the code as it evolves. Hopefully the refactoring should make the flow and responsibilities of `Start()` clear without the comments.